test/pynrn/test_a_neuronoptions.py

```diff
@@ -0,0 +1,43 @@
+# test_a_neuronoptions.py named to be first test file in this folder.
```

Comment:
Maybe we can use CMake to pass on NEURON_MODULE_OPTIONS. Will explore that when I resume work on CI testing.